apps/block_scout_web/lib/block_scout_web/channels/address_channel.ex

```diff
@@ -21,7 +21,7 @@ defmodule BlockScoutWeb.AddressChannel do
     with {:ok, casted_address_hash} <- AddressHash.cast(socket.assigns.address_hash),
          {:ok, address = %{fetched_coin_balance: balance}} when not is_nil(balance) <-
            Chain.hash_to_address(casted_address_hash),
-         exchange_rate <- Market.get_exchange_rate("cGLD") || Token.null(),
+         exchange_rate <- Market.get_exchange_rate("CELO") || Token.null(),
```

Comment:
I think we're fetching exchange rates from the DB and it's still `cGLD` there. See https://github.com/celo-org/blockscout/blob/master/apps/explorer/lib/explorer/chain.ex#L4249 So, I guess we either have to make translation in that method from `CELO` to `cGLD` or change what we save in the database as well.